packages/react-native/React/Fabric/RCTConversions.h

```diff
@@ -52,6 +91,15 @@ inline UIColor *_Nullable RCTUIColorFromSharedColor(const facebook::react::Share
     return [UIColor whiteColor];
   }
 
+  facebook::react::Color color = *sharedColor;
+  switch (color.getKind()) {
+    case facebook::react::DynamicKind:
+      return RCTUIColorFromDynamicColor(color);
+    case facebook::react::UndefinedKind:
+    case facebook::react::ComponentsKind:
+      break;
+  }
```

Comment:
this would read better with 
```objc
if (color.getKind == facebook::react::DynamicKind) {
  return RCTUIColorFromDynamicColor(color);
}
```
It is fewer lines and we are ignoring the other cases nevertheless.